src/gateway/router/routes/sequencerRoute.ts

```diff
@@ -166,7 +166,7 @@ export async function sequencerRoute(ctx: Router.RouterContext) {
     await trx.commit();
     sLogger.info('Total sequencer processing', benchmark.elapsed());
 
-    sendNotificationToCache(ctx, contractTag, undefined, interaction);
+    sendNotification(ctx, contractTag, undefined, interaction);
```

Comment:
does it stay undefined on purpose? the tags for interactions are sent differently?

Reply:
the interaction itself contains the tags